buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidInstrumentationTarget.groovy

```diff
@@ -24,6 +24,9 @@ class AndroidInstrumentationTarget extends AndroidAppTarget {
 
     @Override
     Scope getApt() {
+        if (isKapt) {
+            return Scope.from(project, "kaptAndroidTest")
```

Comment:
This does not handle variant specific configurations correctly

Reply:
https://youtrack.jetbrains.com/issue/KT-23411